docs/howto/manage-a-12-factor-app-charm.rst

```diff
@@ -4,13 +4,14 @@
 Manage a 12-factor app charm
 ============================
 
-    See also: `Juju | 12-factor app charms <https://juju.is/docs/juju/charmed-operator>`_
+    See also:
+    `Juju | 12-factor app charms <https://juju.is/docs/juju/charmed-operator>`_
```

Comment:
If you add Juju to intersphinx like I did in https://github.com/canonical/charmcraft/pull/2095/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3:

```suggestion
    :external+juju:ref:`Juju | Charm <charm>`
```

Reply:
The intersphinx links are nice but it's crucial that we find a way to prefix them with the target documentation set on the pattern you see before the change ("Juju | ..."). Otherwise, given that we have same title stories in multiple documentation sets (a feature, not a bu, as I've argued in the past), the links become very confusing.